styles/style.css

```diff
@@ -0,0 +1,246 @@
+* {
+    margin:0;
+    padding: 0;
+}
+
+.centered {
+    display: flex;
+    justify-content: center;
+}
+
+body {
+    font-family: sans-serif;
+    font-size: 4em;
+}
+
+.parallax_wrapper {
+    height:100vh;
+    overflow-x: hidden;
+    overflow-y: auto;
+    perspective: 300px;
+}
+
+.intro_outro_screen {
+    height: 100vh;
+    align-items: center;
+
+}
+
+.parallax_group {
+    position: relative;
+    height: 100vh;
+    transform-style: preserve-3d;
+}
+
+.parallax_layer {
+    position: absolute;
+    inset: 0;
+}
+
+.top_layer {
+     /* scale calculation: 1 + ((Z transform value * -1) / perspective) */
+    transform: translateZ(110px) scale(0.63);
+}
+
+.mid_layer {
+    transform: translateZ(0);
+}
+
+.base_layer {
+    /* scale calculation: 1 + ((Z transform value * -1) / perspective) */
+    transform: translateZ(-300px) scale(2); 
+}
+
+
+#intro {
+    z-index: 10;
+    background-color: hotpink;
+}
+
+
+/* #group_one {
+    z-index: 5;
+} */
+.group_one_z_index {
+    z-index: 5;
+}
+#group-one > .base_layer {
+    background-color: lightskyblue;
+    
+}
+#group-one > .mid_layer {
+    /* background-color: cyan; opacity: 0.5; */
+}
+#group-one > .top_layer {
+    /* background-color: magenta; opacity: 0.5; */
+}
+
+
+/* #group_two {
+    z-index: 0;
+} */
+/* .group_two_z_index {
+    z-index: 0;
+} */
```

Comment:
Possibly related to the middle section not appearing? But the group 3 stuff is also commented out, but seems to work, so 🤷 . With all the perspective stuff and transforms, I'm not sure what is causing each section to appear vertically stacked. I tried commenting out the invisible middle section, and that caused the third group to get messed up, so there's definitely more research to do here!